src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -1077,6 +1077,22 @@ def get_az_user_agent():
     return ' '.join(agents)
 
 
+def get_az_rest_user_agent():
+    """Get User-Agent for az rest calls"""
```

Comment:
Consider add an example of the final result.

Reply:
sure